contrib/cron-jobs/pyasn

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+
+# How often should this run? 
+#  Once a day max
+
+# crontab entry for this (proposal):
+# # m h  dom mon dow   command
+# 02  01 *   *   *     ( cd /tmp; /my/path/to/this/script )
+# 
+
+path=/usr/local/bin
+export PATH=$PATH:$path
+
+if [ ! -x $path/pyasn_util_download.py ]; 
+  echo "need to have pyasn installed. Exiting"
+  exit 255
+fi
+
+cd /tmp
+mkdir -p pyasn
+cd pyasn
+
+pyasn_util_download.py --latest
+
+latest=$(ls -tr | tail -n 1)
```

Comment:
I'm feeling very uncomfortable.. We could filter for the filename ending at least.